sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs

```diff
@@ -58,7 +58,7 @@ public enum FieldValueType
         Dictionary,
 
         /// <summary>
-        /// Used for <see cref="FieldValueSelectionMark"/> type.
+        /// Used for Selection Mark type.
```

Comment:
This is a tricky one. So `FieldValueSelectionMark` was wrong because that is an internal type. It should be `SelectionMarkState` but saying `Used for SelectionMarkState type` sounded weird for me... so I just said Selection Mark.
I will add `SelectionMarkState` as it is the real type

Reply:
I see. No major concerns from my side, so your call :)